bool color  #敌方颜色 0：红色 1：蓝色
int16 remaining_time  #剩余比赛时间
int16 bule_base_blood  #蓝方基地血量
int16 red_base_blood  #红方基地血量
int16 r1_blood  #红英雄血量
int16 b1_blood  #蓝英雄血量
int16 r2_blood  #红工程血量
int16 b2_blood  #蓝工程血量
int16 r3_blood  #红三血量
int16 b3_blood  #蓝三血量
int16 r4_blood  #红四血量
int16 b4_blood  #蓝四血量
int16 r5_blood  #红五血量
int16 b5_blood  #蓝五血量
int16 r6_blood  #红无人机血量
int16 b6_blood  #蓝无人机血量
int16 r7_blood  #红哨兵血量
int16 b7_blood  #蓝哨兵血量
int16 r_outpost  #红方前哨站血量
int16 b_outpost  #蓝方前哨站血量
bool energy  #是否可击打能量机关 0：否 1：可
bool r_darts  #红方飞镖闸门是否打开 0：关闭 1：打开
bool b_darts  #蓝方飞镖闸门是否打开 0：关闭 1：打开
int16 bullet_dose  #自身弹量
#枪口热量
#弹速
#机器人等级
#